ui/src/components/vault/VaultManagement/NatPurseAmountInput.jsx

```diff
@@ -75,6 +76,8 @@ const PurseAmountInput = ({
           decimalPlaces={decimalPlaces}
           placesToShow={placesToShow}
           disabled={amountInputDisabled}
+          error={error !== null}
+          helperText={error ?? ' '}
```

Comment:
There may be a need for non-error helper text for a different feature, but that can be handled separately when needed.

The ' ' is a hack to prevent the height of the input from changing (and moving around the UI) when the helperText appears and disappears.

Reply:
Is `??` in standard JS yet? Or is that transformed by our bundler? Just curious, I suppose.

Reply:
I think so https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#browser_compatibility